src/transformers/tokenization_roberta_fast.py

```diff
@@ -172,6 +173,32 @@ def __init__(
             **kwargs,
         )
 
+    @property
+    def mask_token(self) -> str:
+        """
+        :obj:`str`: Mask token, to use when training a model with masked-language modeling. Log an error if used while
+        not having been set.
+
+        Roberta tokenizer has a special mask token to be usble in the fill-mask pipeline. The mask token will greedily
+        comprise the space before the `<mask>`.
+        """
+        if self._mask_token is None and self.verbose:
+            logger.error("Using mask_token, but it is not set yet.")
```

Comment:
Do we need the `self.verbose` now that the logger is centralized and can be easily managed?

Reply:
No indeed, let me remove that in a follow-up PR